src/components/LoginForm.tsx

```
@@ -14,22 +14,24 @@ import { useForm } from "react-hook-form";
 import { Dispatch, SetStateAction, useState } from "react";
+import { handleGetEvents } from "@/actions/event-actions";
 
 type LoginFormProps = {
   type: "Login" | "Sign Up";
-  handleLoginSubmit?: (
-    values: { email: string; password: string }
-  ) => Promise<String>;
+  handleLoginSubmit?: (values: {
+    email: string;
+    password: string;
+  }) => Promise<String>;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)